style.css

```diff
@@ -635,6 +720,75 @@ label {
     gap: 50px;
   }
 
+  .modal {
+    width: 50%;
+    background-color: #fff;
+    margin: 10px auto;
+    padding: 1rem;
+    flex-direction: column;
+    height: auto;
+    border-radius: 16px;
+  }
+
+  .d-header {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
On the desktop version, kindly adjust the design to be as close to the design as possible. Check the illustration below for details:

Your design:
![image](https://user-images.githubusercontent.com/20554583/233111988-6c1b80b1-7e65-4b64-9a45-f91e18b17481.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/233111117-73e6c482-da97-44c1-ab98-6881c76396b3.png)